server/service/src/processors/transfer/shipment/create_inbound_shipment.rs

```diff
@@ -78,6 +78,26 @@ impl ShipmentTransferProcessor for CreateInboundShipmentProcessor {
 
         InvoiceRowRepository::new(connection).upsert_one(&new_inbound_shipment)?;
 
+        if outbound_shipment.invoice_row.status.clone() == InvoiceRowStatus::Picked {
```

Comment:
This check is not needed, this action will only fire once:
https://github.com/openmsupply/open-msupply/blob/88774b26d841ece222fba472fffb1e5980ce9fd8/server/service/src/processors/transfer/shipment/create_inbound_shipment.rs#L33-L34